File: PythonStmSrv/Model/Student.py
class Student:
    count = 0

    @classmethod
    def autoIncr(self):
        self.count += 1
        return self.count

    def __init__(self, nume, prenume, grupa):
        self._sID = self.autoIncr()
        self._nume = nume
        self._prenume = prenume
        self._grupa = grupa

    def getSID(self):
        return self._sID

    def setSID(self, sID):
        self._sID = sID

    def getNume(self):
        return self._nume

    def setNume(self, nume):
        self._nume = nume

    def getPrenume(self):
        return self._prenume

    def setPrenume(self, prenume):
        self._prenume = prenume

    def getGrupa(self):
        return self._grupa

    def setGrupa(self, grupa):
        self._grupa = grupa

File: JavaJmsSrv/src/main/java/Model/Student.java
package Model;

import java.util.concurrent.atomic.AtomicInteger;

public class Student {

    private static final AtomicInteger count = new AtomicInteger(0);

    private int sID;
    private String nume;
    private String prenume;
    private String grupa;

    public Student(String nume, String prenume, String grupa) {
        this.sID = count.incrementAndGet();
        this.nume = nume;
        this.prenume = prenume;
        this.grupa = grupa;
    }

    public int getsID() {
        return sID;
    }

    public void setsID(int sID) {
        this.sID = sID;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }

    public String getGrupa() {
        return grupa;
    }

    public void setGrupa(String grupa) {
        this.grupa = grupa;
    }
}
